gap/PolygonalComplexes/properties.gd

```diff
@@ -790,3 +823,42 @@ DeclareOperation( "IsChaoticVertexNC", [IsTwistedPolygonalComplex, IsPosInt] );
 DeclareAttribute( "FaceTwoColouring", IsPolygonalComplex );
 #! @EndGroup
 
+#! @Section Block type
```

Comment:
Do you have any ideas what other methods can be part of this chapter? A section for only one method will not make sense.

Reply:
I'll undo this section until more methods for spheres with 3 waists have been added to the package.